### PR TITLE
docs: edit README file

### LINKED ISSUE #2: docs: edit README file
Edit README file for the repo explaining that all the technical documentation and research would be in this repository. Research files must be added under the `research` folder.

Also mention that for further assistance they can join the following Telegram groups:
Semaphore-Stellar (maintainers of Semaphore Stellar): https://t.me/+-9623JNgGjEyNzI5
Semaphore Community (core developers of the Semaphore protocol): @semaphore_community (search it on Telegram)

### PR DESCRIPTION
**Pull request by mamicho16**
Closes #2

Hello! This PR is ready for review!
Let me know if ererything is good or if I should change something.
I appreciate the opportunity to contribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# semaphore-stellar-docs
\ No newline at end of file
+# semaphore-stellar-docs
+
+Welcome to the Semaphore Stellar documentation repository. This repository serves as the central hub for all technical documentation and research related to the Semaphore Implementation in Stellar.
+
+## Repository Organization
+
+The repository is structured to make finding information as straightforward as possible. All technical documentation is available in the root directory, providing immediate access to essential implementation details, guides, and specifications.
```
Comment (evgongora) [inline README.md:17]: Could you please redirect to the contracts repo?

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+## Community Support
+
+We have an active community ready to help! For additional support or to engage with our team, you can join our Telegram groups:
+
+### Semaphore Stellar
+- If you're interested in discussing with Semaphore Stellar maintainers directly
+- Join: https://t.me/+-9623JNgGjEyNzI5
+
+### Semaphore Community
```
Comment (evgongora) [inline README.md:30]: Try to add a comment that we're not linked to Semaphore Core team but that the group can be used for direct support.

**Comment by evgongora** [comment]
Gj so far @mamicho16 , could you please also add some information about Semaphore and why we're making this project in Stellar

**Comment by mamicho16** [comment]
I have applied all the requested changes:
- I added information about Semaphore and the project's purpose in Stellar
- I added the link to the contracts repository in the documentation section
- I updated the description about the Semaphore Community group to clarify its role

Could you please review these modifications to ensure they fullfil your requirements? Thank you for your guidance.

File: README.md [hunk 3]
```diff
@@ -1 +1,32 @@
...
+For research-related content, we maintain a dedicated `research` folder. This folder contains all research files and related materials. When contributing new research, please ensure your files are added to this folder to maintain our organizational structure.
+
+## Community Support
+
+We have an active community ready to help! For additional support or to engage with our team, you can join our Telegram groups:
+
+### Semaphore Stellar
+- If you're interested in discussing with Semaphore Stellar maintainers directly
+- Join: https://t.me/+-9623JNgGjEyNzI5
+Note: Semaphore Community group is not directly affiliated with the Semaphore Core team, it serves as an excellent resource for direct support and community engagement.
```
Comment (evgongora) [inline README.md]: Can you change Semaphore community to Semaphore Stellar group?

**Comment by evgongora** [comment]
Perfect @mamicho16 just one little change

**Comment by mamicho16** [comment]
I changed the name of the group, let me know if you need something else.

**Review by bitfalt — APPROVED** [review]
lgtm! Nice work.

**Pull request merged**